client/src/Views/Reports.vue

```diff
@@ -48,10 +49,11 @@ import {
 } from "../shared/formatters";
 import Table from "../components/Table";
 import FillChart from "../components/FillChart";
+import RadarChart from "../components/RadarChart";
 
 export default {
   name: "Reports",
-  components: { Table, FillChart },
+  components: { Table, FillChart, RadarChart },
   data() {
     return {
```

Comment:
Just FYI, this RadarChart will need to go in the new AllShouts component when my PR goes up